### ISSUE TITLE
Template/Slots displaying incorrect count

### ISSUE THREAD

**Issue by marcothesis** [remnoteio/remnote-issues#470]
**Describe the bug**
I use templates utilizing the slots feature but it one of my templates, the count for 2 different slots appear to be linked together and displayed the sum of both slots. Remnote does not seem to be able to refresh or update this ‘count’ based on the individual slot properly.

**To Reproduce**
1. When in my ‘Metadata’ template, count on ‘Status’ slot is 773, compared to the other slots at 387.
2. Upon clicking on the 773 to pull up a search portal listing those results, it looks like ‘Status’ slot is linked and added together with ‘Full text’ slot.

**Expected behavior**
The ‘Status’ slot should count 387 like the other slots in the template.

**Screenshots**
![Screenshot 2021-01-04 at 2 05 32 PM](https://user-images.githubusercontent.com/10230857/103506290-53546480-4e97-11eb-8a48-1ba1d197501c.png)

**Desktop (please complete the following information):**
 - OS: macOS Big Sur, Remnote Desktop App
 - Version: Remnote Desktop App - 1.0.38 (1.0.38.183)

**Additional context**

**Comment by marcothesis** [comment]
A follow-up with a similar issue as they both involved tag references that are tabulated wrongly.

Tag Reference below surface incorrectly in portals. The tag #Read below clearly should not be in the results and when I zoom into the specific Rem of "What Happened in 2020", I also could not find any tags of #Processing that would have allow it to appear there.

![example](https://user-images.githubusercontent.com/10230857/105164789-87df4600-5b50-11eb-8312-3beb4051a7ac.png)

**Comment by drkvj** [comment]
@marcothesis what's the status of ths? HAve you tried to delete the read tag? what does it do?

**Comment by marcothesis** [comment]
> @marcothesis what's the status of ths? HAve you tried to delete the read tag? what does it do?

I guess nothing has been done since it was reported? When I delete the 'read' tag and this particular 'status' slot has no tag now, yet the slot is still being captured by the 'processing' search portal. It's like the slot has an 'invisible' tag of processing to cause it to still appear here.

When I looked into the search portal for 'Status', there's a lot of redundancy like these and it even captures other slots which has no relevant tags in it.

![status](https://user-images.githubusercontent.com/10230857/115098781-83b2a100-9f64-11eb-9a9a-859348b6a301.png)

So the slots are corrupted, there is no fix yet and there is no bulk processing feature where I can mass edit or rather mass delete these corrupted slots and insert new ones.

**Comment by marcothesis** [comment]
I understand that this bug might be account-specific and hard to pin down with a fix. What I would appreciate is a way to at least manually re-process and fix this myself in an efficient manner, through a combination of advanced queries and batch selection/edit of rems?

**Comment by drkvj** [comment]
Yes, I completely agree that batch editing of rems can certainly help in fixing many of the account specific problems until we achieve stability that they don't happen anymore. I will beam this up to the team with higher priority.
